Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM debian
+FROM alpine:3.5
```

Comment:
i'm glad you've had a good experience with it :) we've had mixed results. i like the idea of a smaller image, but would like us to preserve glibc until this undergoes a more substantial stress test